src/faker/faq.ts

```diff
@@ -0,0 +1,93 @@
+const dataAccordions = [
+  {
+    name: 'Shipping Information',
+    data: [
+      {
+        title: 'How will my parcel be deliverd?',
```

Comment:
title, description vẫn quá đơn giản
thêm cho anh các trường hợp sau:
text dài trên 300 chữ, 500 chữ, 1000 chữ
chứa emoji 🤭🤥🤭🤟🫶😫👮‍♂️ (https://emojicopy.com/)
chứa thẻ <h1>Vd title đậm</h1>
-> đảm bảo không được cắt content, phải hiển thị đủ, tag thẻ html thì phải hiển thị dưới dạng full text (tức là vãn hiển thị <h1>... chứ ko phải hiện chữ in đậm)